admin/herbert_on.m.template

```diff
@@ -1,55 +1,51 @@
-function path=herbert_on(non_default_path)
+function path = herbert_on(non_default_path)
 % The function intended to switch Herbert on and
-% return the path were Herbert is resided or 
+% return the path were Herbert is resided or
 % empty string if Herbert has not been found
 %
 %
-% The function has to be present in Matlab search path 
+% The function has to be present in Matlab search path
 % and modified for each machine to know default Herbert location
 %
 % The default Herbert location is the place where Herbert_init script
 % can be found.
 %
 %Usage:
-%>>path=herbert_on(); 
+%>>path=herbert_on();
 %       enables Herbert and initiates Herbert default search path
-%>>path=herbert_on('where'); 
+%>>path=herbert_on('where');
 %       reports current location of Herbert or empty if not found
 %
-%>>path=herbert_on('a path'); 
+%>>path=herbert_on('a path');
 %       initiates Herbert on non-default search path
-%
-%
-%
-%
-her_default_path='/usr/local/mprogs/Herbert';
-%
+
+default_herbert_path = '/usr/local/mprogs/Herbert';
+
 if exist('non_default_path','var') && (strcmpi(non_default_path,'where') || strcmpi(non_default_path,'which'))
-    path = find_her_default_path(her_default_path);   
+    path = find_herbert_path(default_herbert_path);
```

Comment:
this code-path never calls `herbert_init`

Reply:
It's not supposed to call `herbert_init`. It's just telling you where - if anywhere - Herbert is currently activated